src/style.css

```
@@ -1 +1,38 @@
 /* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-image: url(./images/star.svg);
+}
+
+.stars--1 .stars__star:nth-child(-n+1) {
+  background-image: url('./images/star-active.svg');
+}
+
+.stars--2 .stars__star:nth-child(-n+2) {
+  background-image: url('./images/star-active.svg');
+}
+
+.stars--3 .stars__star:nth-child(-n+3) {
+  background-image: url('./images/star-active.svg');
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url('./images/star-active.svg');
+}
+
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url('./images/star-active.svg');
+}
```

```suggestion
.stars--1 .stars__star:nth-child(-n+1),
.stars--2 .stars__star:nth-child(-n+2),
.stars--3 .stars__star:nth-child(-n+3),
.stars--4 .stars__star:nth-child(-n+4),
.stars--5 .stars__star:nth-child(-n+5)
{
  background-image: url(./images/star-active.svg);
}
```
you can shorten this